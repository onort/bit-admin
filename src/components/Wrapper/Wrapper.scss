@import "../../styles/main.scss";

.wrapper {
  height: 100vh;
  width: 100vw;
  background-color: $lighter-bg;
  display: flex;
}
