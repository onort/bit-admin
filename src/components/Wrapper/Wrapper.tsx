import React from "react"
import cx from "classnames"

import styles from "./Wrapper.scss"

interface Props {
  className?: string
}

const Wrapper: React.SFC<Props> = props => (
  <div className={cx(styles.wrapper, props.className)}>{props.children}</div>
)

export default Wrapper
