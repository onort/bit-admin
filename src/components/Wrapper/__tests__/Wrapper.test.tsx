import React from "react"
import { shallow } from "enzyme"

import Wrapper from "../"

const testClassName = "testClassName"

describe("<Wrapper />", () => {
  it("should match snapshot", () => {
    const wrapper = shallow(<Wrapper />)
    expect(wrapper).toMatchSnapshot()
    expect(wrapper.html()).toMatchSnapshot()
  })

  it("should have a custom className", () => {
    const wrapper = shallow(<Wrapper className={testClassName} />)
    expect(wrapper.find(".wrapper").hasClass(testClassName)).toBe(true)
  })
})
