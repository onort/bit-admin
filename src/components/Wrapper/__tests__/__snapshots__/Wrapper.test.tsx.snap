// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wrapper /> should match snapshot 1`] = `
<div
  className="wrapper"
/>
`;

exports[`<Wrapper /> should match snapshot 2`] = `"<div class=\\"wrapper\\"></div>"`;
